traitsui/qt4/text_editor.py

```diff
@@ -72,16 +77,20 @@ def init(self, parent):
         if factory.password:
             control.setEchoMode(QtGui.QLineEdit.Password)
 
-        if factory.auto_set and not factory.is_grid_cell:
-            if wtype == QtGui.QTextEdit:
-                control.textChanged.connect(self.update_object)
-            else:
-                control.textEdited.connect(self.update_object)
-
+        self._signals = []
```

Comment:
Why is this line in the `init` needed? Wouldn't `_signals` be an empty list without it (as per default `List` value)?

Reply:
Not needed. It was code evolution. Will remove.